resources/assets/less/app/kanban.less

```diff
@@ -54,6 +54,110 @@
   }
 }
 
+
+/* navpill */
+.kanban-nav {
+	position: relative;
+	margin: 0px 0;
+}
+.kanban-pill-tag {
+	color: #afafaf;
+	padding-left: 10px !important;
+	padding-right: 10px !important;
+}
+.kanban-nav ul {
+	margin: 0;
+	padding: 0;
+}
+.kanban-nav li {
+	margin: 0 5px 10px 0;
+	padding: 0;
+	list-style: none;
+	display: inline-block;
+}
+.kanban-nav a {
```

Comment:
For all of the new changes in this file. Make use to LESS features instead of plain CSS.

For example instead of:
```
.kanban-nav li {
}
.kanban-nav a {
}
```
Make it
```
.kanban-nav  {
   li {
      a {

      }
   }
}
```